Marlin/Configuration_tenlog.h

```diff
@@ -360,16 +361,17 @@
   #define E0_DRIVER_TYPE DriverType
   #define E1_DRIVER_TYPE DriverType
 #endif
-
-#define LUX_REFERENCE_Y_MIN_POS 0
+#ifndef LUX_REFERENCE_Y_MIN_POS
```

Comment:
We no longer want to code things in this style.  A variable should not be defined if it will be undefined later.  Rather a default variable should be defined in the last #else section (see steps per mm in stock extruder section)